### ISSUE TITLE
关于QwQ-32B训练参数

### ISSUE THREAD

**Issue by amoyplane** [hiyouga/EasyR1#169]
我正在尝试使用8*H20（96GB VRAM）对QwQ-32B进行训练。

不开启cpu_offload时会有显存不够用的问题，开启cpu_offload后内存使用超过了我们机器内存（我们内存只有512G）。

看到README有提到 GRPO Full Fine-Tuning 32B 预估是 8*80GB，不知道是否方便提供32B模型的训练参数参考呢？

我们使用的训练参数如下：

> data:
>   train_files: hiyouga/math12k@train
>   val_files: hiyouga/math12k@test
>   prompt_key: problem
>   answer_key: answer
>   image_key: images
>   max_prompt_length: 2048
>   max_response_length: 2048
>   rollout_batch_size: 512
>   shuffle: true
>   seed: 1
> 
> algorithm:
>   adv_estimator: grpo
>   use_kl_loss: true
>   kl_penalty: low_var_kl
>   kl_coef: 1.0e-2
> 
> worker:
>   actor:
>     global_batch_size: 128
>     micro_batch_size_per_device_for_update: 4
>     micro_batch_size_per_device_for_experience: 16
>     max_grad_norm: 1.0
>     padding_free: true
>     ulysses_sequence_parallel_size: 1
>     model:
>       model_path: /home/admin/pretrained-model/QwQ-32B/
>       enable_gradient_checkpointing: true
>       trust_remote_code: true
>       freeze_vision_tower: false
>     optim:
>       lr: 1.0e-6
>       weight_decay: 1.0e-2
>       lr_warmup_ratio: 0.0
>     fsdp:
>       enable_full_shard: true
>       enable_cpu_offload: false
>       enable_rank0_init: true
>     offload:
>       offload_params: true
>       offload_optimizer: true
> 
>   rollout:
>     temperature: 1.0
>     n: 5
>     gpu_memory_utilization: 0.6
>     enforce_eager: false
>     enable_chunked_prefill: false
>     tensor_parallel_size: 2
>     limit_images: 0
>     val_override_config:
>       temperature: 0.5
>       n: 1
> 
>   ref:
>     fsdp:
>       enable_full_shard: true
>       enable_cpu_offload: true
>       enable_rank0_init: true
>     offload:
>       offload_params: false
> 
>   reward:
>     reward_type: function
>     compute_score: math
> 
> trainer:
>   total_episodes: 15
>   #logger: ["console"]
>   logger: ["console"]
>   project_name: test_qwq
>   experiment_name: test_qwq_32b_grpo
>   n_gpus_per_node: 8
>   nnodes: 1
>   report_kl: true
>   val_freq: 5
>   val_before_train: true
>   val_only: false
>   val_generations_to_log: 1
>   save_freq: 10
>   save_limit: 3
>   save_checkpoint_path: null
>   load_checkpoint_path: null

**Comment by hiyouga** [comment]
https://github.com/hiyouga/EasyR1/blob/main/examples/qwen2_5_vl_32b_geo3k_grpo.sh

**Comment by lx709** [comment]
> https://github.com/hiyouga/EasyR1/blob/main/examples/qwen2_5_vl_32b_geo3k_grpo.sh

用了这个配置，开启cpu_offload后内存依然超过了机器内存 （GH200， 864GB memory）
